assets/i18n/active.en.toml

```diff
@@ -18,12 +18,16 @@ Example = "Example"
 On = "on"
 As = "as"
 Of = "of"
+To = "Go to"
+For = "for"
 
 # Layout frame
 ToMain = "To main content"
 Logo = "PDOK Logo: Go to landing page"
+SwitchLanguage = "Zet taal op"
```

Comment:
dit is nederlands omdat het dan voor nederlands die perongeluk in de engelse variant zitten dit kunnen lezen?

Reply:
Ja, de items gerelateerd aan de taal switch zijn dus precies omgekeerd. Nederlands is de standaard, de link is dan dus 'English' met uiteindelijke ARIA label 'Set language to English'; en vice versa als de taal op Engels staat.

Reply:
Zou "Wijzig taal naar" duidelijker vinden